pyro/distributions/transforms/householder.py

```diff
@@ -105,7 +105,14 @@ def _inverse(self, y):
         point around a plane, then the same operation will reflect it back
         """
 
-        return self._call(y)
+        x = y
+        u = self.u()
+        for idx in range(self.count_transforms):
+            # NOTE: Need to apply transforms in reverse order as forward operation!
+            jdx = self.count_transforms - idx - 1
```

Comment:
use `reversed()` instead of explicit indexing?

Reply:
was this not exercised by previous tests?

Reply:
No it wasn't, but it is now. And I've made that change!